src/rust/common/src/bytes.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 Twitter, Inc.
```

Comment:
2022 ?

Reply:
oh just copied it from another file. I've actually never added a license header and don't really know what the practice is

Reply:
New modules/crates/... should have a license header that reflects the current year.